source/common/common/fancy_logger.cc

```diff
@@ -0,0 +1,118 @@
+#include <atomic>
+#include <memory>
+
+#include "common/common/logger.h"
+
+using spdlog::level::level_enum;
+
+namespace Envoy {
+
+/**
+ * FancyLogger implementation: a logger with fine-grained log control.
+ */
+
+/**
+ * If the sink is initialized.
+ */
+int kSinkInit = 0;
```

Comment:
This is not used.

This is a variable with a constant name. Better change it.

Reply:
Addressed.